Tests/UnitTests/Mocks/MockBackend.swift

```diff
@@ -34,7 +34,7 @@ class MockBackend: Backend {
         let systemInfo = try! MockSystemInfo(platformInfo: nil, finishTransactions: false, dangerousSettings: nil)
         let attributionFetcher = AttributionFetcher(attributionFactory: MockAttributionTypeFactory(),
                                                     systemInfo: systemInfo)
-        self.init(httpClient: MockHTTPClient(systemInfo: systemInfo, eTagManager: MockETagManager()),
+        self.init(httpClient: MockHTTPClient(systemInfo: systemInfo, eTagManager: MockETagManager(), timeoutSeconds: 7),
```

Comment:
Looks like all `MockHTTPClient` are using the same `7`. Consider that the value likely doesn't matter since it doesn't do real requests, maybe move that value to the constructor so we don't need this magic number in all the tests that use it?

Reply:
✅